Trees/FenwickTreeImplementation.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Author: Mohit Kumar
+ * Fedwick Tree Implementation in JavaScript
+ * Fedwick Tree Implementation for finding prefix sum.
+*/
+
+class FenwickTree {
+  constructFenwickTree (feneickArray, array, n) {
```

Comment:
Rename this function to `constructor` so that is can be called during initialization,

Reply:
Renamed constructFenwickTree as a constructor.
Made required changes in FenwickTreeImplementation.test.js